HISTORY.md

```diff
@@ -52,6 +52,8 @@
   * Change neural network types to avoid unnecessary use of rvalue references
     (#2259).
 
+  * Add Hinge-Embedding-Loss Function (#2229).
```

Comment:
I don't think there needs to be hyphens here

Reply:
Thanks for noticing, I have changed it in the new commit Kindly have a look.
Thanks.